### PR TITLE
Removing deprecated Storage library

### PR DESCRIPTION
**Pull request by serkantkaraca**
- Moving to Microsoft.Azure.Storage since 9.3.0 library is deprecated recently.
- Up version to 4.3.0

### CODE REVIEW

**Review by JamesBirdsall — APPROVED** [review]

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 4.3.0 (Unreleased)
+
+
 ## 4.2.1 (Unreleased)
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md:6]: If 4.2.1 went unreleased, is there a reason that we'd leave the empty changelog?

File: sdk/eventhub/Microsoft.Azure.EventHubs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 4.3.0 (Unreleased)
+
+
 ## 4.2.1 (Unreleased)
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs/CHANGELOG.md:6]: Same question as with EPH.

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault.Core" />
-    <PackageReference Include="WindowsAzure.Storage" />
```
Comment (jsquire) [inline sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj:21]: I wonder if we can remove this from the central package versions as well.   I'll follow-up on this.

**Review by jsquire — APPROVED** [review]

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md (hunk 1 above)
Comment (serkantkaraca) [reply sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md:6]: 4.2.1 was a space holder for testing purpose only. I will remove it before the release.

File: sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj (hunk 3 above)
Comment (serkantkaraca) [reply sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/Microsoft.Azure.EventHubs.Processor.csproj:21]: Thx Jesse

**Pull request merged**